src/code/HTTPRequestHandler/HTTPRequestHandler.cpp

```diff
@@ -4,8 +4,46 @@
 
 void HTTPRequestHandler::GET(HTTPCall &request)
 {
-	(void)request;
-	request.setResponse("HTTP/1.1 200 OK\r\n\r\nYOU SENT A GET REQUEST");
+	try
+	{
+		BasicHTTPRequest requestRecived = request.getBasicRequest();
```

Comment:
i think HTTPCall have it's own BasicHTTPRequest. no need to create here a new one